### PR TITLE
Popup api task

### PR DESCRIPTION
**Pull request by Conversee12**
### HTML&CSS_Popup

[Demo](https://conversee12.github.io/Hooli-style-Popup/) |
[Code base](https://github.com/Conversee12/Hooli-style-Popup.git)

The code is submitted in a dedicated feature branch.

Only code files are submitted.

Please, review.

### CODE REVIEW

**Comment by Conversee12** [comment]
I fixed space at the end files. I didn't see common mistakes, please, let me know if something needs to be changed.

File: submissions/Conversee12/HTML&CSS Popup/style.css [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+
+.more__burger {
+  display: none;
+}
+
+.header__more:checked~.more__burger {
+  display: block;
+}
+
+
```
Comment (artem-trubin) [inline submissions/Conversee12/HTML&CSS Popup/style.css:235]: Two empty lines.

File: submissions/Conversee12/HTML&CSS Popup/style.css [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+  justify-items: center;
+}
+
+.header__burger-list li {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  align-self: center;
+
```
Comment (artem-trubin) [inline submissions/Conversee12/HTML&CSS Popup/style.css:150]: Empty line.

**Review by artem-trubin — CHANGES_REQUESTED** [review]
Hi! Great job. Let's improve it a little.

Your `More` button doesn't have any visual reaction when you get to it by going through all elements with the keyboard. Also, it should disappear when the `More` menu is opened.

Another thing, please be consistent with your code styling, make equal amounts of empty lines between code block, etc.

**Comment by artem-trubin** [comment]
Well done!

**Pull request merged**